asm-patch/Kptfile

```diff
@@ -93,6 +99,12 @@ openAPI:
         setter:
           name: anthos.servicemesh.tag
           value: 1.6.8-asm.9
+    io.k8s.cli.setters.anthos.servicemesh.hubMembershipID:
+      type: string
+      x-k8s-cli:
+        setter:
+          name: anthos.servicemesh.hubMembershipID
```

Comment:
it's under `asm-patch/Kptfile`. Would it make sense to make the file more specific like asm-hub-patch/Kptfile?

Reply:
@elfinhe shall we create a different folder for Hub?

Reply:
We copy pkg/Kptfile to each package to keep them in sync, also to avoid inconsistent setter names. It is okay to have them here.